tests/test_stack_node.py

```diff
@@ -2,42 +2,38 @@
 
 from majorkirby import GlobalConfigNode, StackNode
 from troposphere import Output
-from moto import mock_cloudformation
+from moto import mock_cloudformation_deprecated
```

Comment:
May be the most interesting bit of the PR. I almost went down the route of converting all Boto usage to Boto3, but discovered this deprecated functionality that continues to mock Boto (what we're using).